### PR TITLE
3.1 Search results loading page

### PR DESCRIPTION
**Pull request by BoosterTech**
- Loading page available under the link: **http://localhost:{hostPort}/MovieBrowser#/MovieBrowser/searching**
- Loading style adjusted to Figma style on MovieList & PeopleList pages

### CODE REVIEW

File: src/common/Loader/styled.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+100%{transform:rotate(360deg);
+}`;
+
+export const StyledSvg = styled.svg`
+  display: block;
+  max-width: 91px;
+  max-height: 91px;
+  margin: auto;
+  margin-top: 219px;
+  fill: #f5f5fa;
```
Comment (Arkadiusz0809) [inline src/common/Loader/styled.js:15]: we use theme features

File: src/common/SearchPage/styled.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { styled } from "styled-components";
+
+export const MessageHeader = styled.h1`
+  width: 602px;
+  height: 43px;
+  font-size: 36px;
+  font-weight: 600;
+  line-height: 43.2px;
+  text-align: left;
+  color: black;
```
Comment (Arkadiusz0809) [inline src/common/SearchPage/styled.js:10]: we use theme features

**Review by Arkadiusz0809 — COMMENTED** [review]
does it work?

File: src/common/Loader/styled.js (hunk 1 above)
Comment (BoosterTech) [reply src/common/Loader/styled.js:15]: What do you mean you use them features. these are styles for the loader spinner

File: src/common/SearchPage/styled.js (hunk 2 above)
Comment (BoosterTech) [reply src/common/SearchPage/styled.js:10]: yeah. me too. I am front end developer too hehe

File: src/common/Loader/styled.js (hunk 1 above)
Comment (BoosterTech) [reply src/common/Loader/styled.js:15]: Yeah, works perfectly

**Pull request merged**